src/Sulu/Component/Content/Compat/Structure/StructureBridge.php

```diff
@@ -124,7 +120,7 @@ public function getWebspaceKey()
      */
     public function getUuid()
     {
-        return $this->getDocument()->getUuid();
+        return $this->getDocument()->getUuid($this->document);
```

Comment:
i think this call cannot work before! the required argument `$document` was not given ...